charts/fluentd/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.networkPolicy.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ include "fluentd.fullname" . }}
+  labels:
+    {{- include "fluentd.labels" . | nindent 4 }}
+spec:
+  podSelector:
+    matchLabels:
+      {{- include "fluentd.selectorLabels" . | nindent 6 }}
+  ingress:
+    - ports:
+      - port: 24231
+        protocol: TCP
+      {{- if .Values.service.ports }}
+      {{- range $port := .Values.service.ports }}
+      - port: {{ $port.containerPort }}
+        protocol: {{ $port.protocol }}
+      {{- end }}
+      {{- end }}
+  {{- with .Values.networkPolicy }}
```

Comment:
what happens if I want to include other IngressPolicies?

Can you allow user to set ingress and egress policies form the values file and this policies being templated here?